google_sheets/google_sheets.py

```diff
@@ -0,0 +1,34 @@
+import httplib2
+
+from apiclient import discovery
+from google_auth import get_credentials
+
+
+def to_sheet(spreadsheet_id, values, sheet_name='Sheet1', row_offset=0, clear_columns=False, app_name=''):
+    credentials = get_credentials(app_name)
```

Comment:
since get_credentials is now offered via the google_auth module we cannot offer a to_sheet function that does not provide a way to declare scopes. That is something an author must be able to change at will.

I recommend adding an optional kwarg that defaults to the google_auth module SCOPES value (do not copy paste the values).

we should also make that variable exported in the google auth library so it can be customized by any author.

This will need to be replicated with the to_csv_file function as well.

These modules are so tightly coupled I must again recommend that we merge them and add an auth.py to this module. That way we can set a default scope that satisfies all these module functions and change one module rather than 2 when we add more functionality that requires more scopes.

We can make maintenance simpler and make more durable software.